en/configure-storage-class.md

```diff
@@ -147,7 +147,7 @@ If the components such as monitoring, TiDB Binlog, and `tidb-backup` use local d
 
     >**Note:**
     >
-    > The number of directories you create depends on the planned number of TiDB clusters, the number of Pumps in each cluster, and your backup method. For each directory, a corresponding PV will be created. Each Pump uses one PV and each Drainer uses one PV. Each [Ad-hoc full backup](backup-and-restore-using-helm-charts.md#ad-hoc-full-backup) task uses one PV, and all [scheduled full backup](backup-and-restore-using-helm-charts.md#scheduled-full-backup) tasks share one PV.
+    > The number of directories you create depends on the planned number of TiDB clusters, the number of Pumps in each cluster, and your backup method. For each directory, a corresponding PV will be created. Each Pump uses one PV and each Drainer uses one PV. All [Ad-hoc full backup](backup-to-aws-s3-using-br.md#ad-hoc-backup) tasks and all [scheduled full backup](backup-to-aws-s3-using-br.md#scheduled-full-backup) tasks share one PV.
```

Comment:
```suggestion
    > The number of directories you create depends on the planned number of TiDB clusters, the number of Pumps in each cluster, and your backup method. For each directory, a corresponding PV will be created. Each Pump uses one PV and each Drainer uses one PV. All [Ad-hoc full backup](backup-to-s3.md#ad-hoc-backup) tasks and all [scheduled full backup](backup-to-s3.md#scheduled-full-backup) tasks share one PV.
```

Reply:
Please also help update the Chinese version.

Reply:
fixed in ae1d343